bepasty/tests/conftest.py

```diff
@@ -0,0 +1,34 @@
+from os import close, unlink
+from random import random
+from tempfile import mkstemp
+
+import pytest
+
+from bepasty.app import create_app, create_storage
+
+
+@pytest.fixture(scope='module')
+def testclient(request):
+    '''
+    creates a Flask-testclient instance for bepasty
+    '''
+    app = create_app()
+    db_file, app.config['DATABASE'] = mkstemp()
+    # reset default permissions
+    app.config['DEFAULT_PERMISSIONS'] = ''
+    # setup a secret key
+    app.config['SECRET_KEY'] = str(random())
+    # setup permissions
+    app.config['PERMISSIONS'] = {
+        'l': 'list',
+        'c': 'create',
+        'r': 'read',
+        'd': 'delete',
+        'a': 'admin'
+    }
```

Comment:
I already wrote one [permission-test](https://github.com/Thor77/bepasty-server/blob/permission_tests/bepasty/tests/test_permissions.py) and just wanted to have a simple way to test the permissions one-by-one, it wouldn't be a big problem to use permissions with incrementing power, but i just didn't need it.